docs/source/getting_started/installation.rst

```diff
@@ -58,13 +58,41 @@ You can install vLLM using pip:
         $ # export VLLM_COMMIT=...
         $ # pip install https://vllm-wheels.s3.us-west-2.amazonaws.com/${VLLM_COMMIT}/vllm-${VLLM_VERSION}-cp38-abi3-manylinux1_x86_64.whl
 
+Build from source (without compilation)
+---------------------------------------
+
+If you want to develop vLLM, and you only need to change the Python code, you can build vLLM without compilation.
+
+The first step is to follow the previous instructions to install the latest vLLM wheel:
+
+.. code-block:: console
+
+    $ export VLLM_VERSION=0.6.1.post1
+    $ pip install https://vllm-wheels.s3.us-west-2.amazonaws.com/nightly/vllm-${VLLM_VERSION}-cp38-abi3-manylinux1_x86_64.whl
```

Comment:
I think we could find a way to make this a little easier -- either by wrapping this in a script that's version invariant, or maybe we could always upload the latest wheel to `https://vllm-wheels.s3.us-west-2.amazonaws.com/nightly/vllm-latest-cp38-abi3-manylinux1_x86_64.whl`

Reply:
cc @simon-mo @khluu if this is possible

Reply:
maybe push to `nightly` during the scheduled build at 9PM daily? or we can push to `latest/vllm-{VLLM_VERSION}...` for every commit

Reply:
Ideally we shouldn't have to update this when we put a new version out, and ideally users could have one command that always. works.

Isn't `export VLLM_VERSION=0.6.1.post1` already out of date, btw?

Reply:
lastest is always pushed. but the version string will always be there. a thing I have known worked in the pass is to set dev versions something like v1.0.0.dev0

Reply:
can we omit the version in the wheel name? i.e. a unified link that always points to the latest wheel. an url re-direction should work I think.